build.sbt

```diff
@@ -14,7 +14,9 @@ lazy val root = (project in file("."))
   .settings(majorVersion := 0)
   .settings(ThisBuild / useSuperShell := false)
   .settings(
-    scalaVersion := "2.13.8",
+    ThisBuild / scalaVersion := "2.13.12",
+  )
```

Comment:
This doesn't need to be in the `.settings`, it can go in the global scope of build.sbt
https://github.com/hmrc/init-service/blob/1770cd0b31e9b5333dc15edf7d5081d7f416183a/init_service/templates/service/build.sbt#L4